src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -183,27 +183,27 @@
                                     {% if message.htmlBody %}
                                         {% set htmlBody = message.htmlBody() %}
                                         <div class="tab">
-                                            <h3 class="tab-title">HTML content</h3>
+                                            <h3 class="tab-title">HTML preview</h3>
                                             <div class="tab-content">
                                                 <pre class="prewrap" style="max-height: 600px">
```

Comment:
This `<pre>` around the iframe does not make sense to me for the HTML preview tab.

Reply:
Removed